### DISCUSSION TITLE
Migrate the entire Managed environment to a Saas environment

### DISCUSSION THREAD

**Discussion by tsoft-globalproject** [Dynatrace/dynatrace-configuration-as-code#268] [Q&A] [answered]
Hello,
We have a challenge, migrating the entire managed environment to the cloud (Saas environment) and I wanted to ask you if this tool could help us take all or part of the configurations.
Please your answers.
Thanks.

**Comment by tatomir146** [comment, marked as answer]
Hi @ConsultandJr, I moved your issue to discussions. Yes, you can use download functionalities to download and then redeploy some of the configuration. Supported APIs are documented here: https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/blob/main/README.md#configuration-types--apis and in case some APIs are missing, here is documentation in how to add these: https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/blob/main/docs/how-to-add-a-new-api.md
Download functionality is a new feature and it is possible you need to optimise some configs before applying to the SaaS env.
